mdstream/mdstream.go

```diff
@@ -21,7 +21,6 @@ import (
 
 const (
 	// mdstream IDs
-	IDProposalGeneral      = 0
```

Comment:
If we remove this it should be marked Invalid.

Reply:
Fixed.  This entire package will be removed once cms migrates to the tstore backend.

https://github.com/lukebp/politeia/blob/tlog/mdstream/mdstream.go#L24